src/System.Data.SqlClient/tests/ManualTests/SQL/AsyncTest/AsyncTest.cs

```diff
@@ -36,7 +37,7 @@ public static void ExecuteTest()
             con.Close();
         }
 
-        [CheckConnStrSetupFact]
+        //[CheckConnStrSetupFact]
```

Comment:
You may want to clean this method and the one above. Looks like they are not used and the [Fact] attribute has been removed